src/reusableComponents/Button.jsx

```diff
@@ -0,0 +1,12 @@
+export const Button = ({ label, className, imageSrc, imgClass }) => {
+
+  return (
+    <button className={className} >
+      <div className="button-content">
+      {/* If imageSrc exists, renders the image */}
+        {imageSrc && <img src={imageSrc} alt={label} className={imgClass} />}  
```

Comment:
Sweet with conditional rendering (: Is there a reason this isn't a styled component? Are there too many different types of buttons?
In my code I was exactly opposite to you 🙈 my only styled component was the buttons!